apps/web/pages/api/book/event.ts

```diff
@@ -214,6 +215,9 @@ const getEventTypesFromDB = async (eventTypeId: number) => {
 type User = Prisma.UserGetPayload<typeof userSelect>;
 
 async function handler(req: NextApiRequest) {
+  const session = await getSession({ req });
+  const currentUser = session?.user;
```

Comment:
I'm using session here so if the user that it's triggering the reschedule of an opt-in booking it's the host it should be Accepted as a status and no pending since he has chosen a new time for the booking.